pangeo-notebook/packages.txt

```diff
@@ -674,65 +704,74 @@ uc-micro-py==1.0.3
 ucx==1.18.0
 ujson==5.10.0
 unicodedata2==16.0.0
+universal_pathlib==0.2.6
 uri-template==1.3.0
 uriparser==0.9.8
 urllib3==1.26.19
 uvicorn==0.34.0
 uvicorn-standard==0.34.0
 uvloop==0.21.0
+virtualizarr==1.2.0
```

Comment:
Virtualizarr being pulled in now from somewhere, so no need #602?

Reply:
Yeah, [`earthaccess=0.13.0`](https://github.com/nsidc/earthaccess/releases/tag/v0.13.0) now depends on virtualizarr, but pins to zarr v2 as mentioned at https://github.com/pangeo-data/pangeo-docker-images/pull/603#discussion_r2001779891 :upside_down_face:

Reply:
I opened this issue to hopefully sort things out https://github.com/nsidc/earthaccess/issues/974